libevmasm/SemanticInformation.cpp

```diff
@@ -351,6 +351,8 @@ bool SemanticInformation::invalidInPureFunctions(Instruction _instruction)
 	case Instruction::ORIGIN:
```

Comment:
This list could use a sort by the way... it's neither alphabetic nor opcode value... but I didn't do it here, because that'll only create conflicts later :-).